resources/config.py

```diff
@@ -143,19 +160,22 @@ def get(self, identifier):
                 description: Config information and contents
                 content:
                   application/json:
-                    schema: ConfigShowSchema
+                    schema: ConfigItemResponseSchema
             404:
                 description: |
                     When config doesn't exist, object is not a config or user doesn't have access to this object.
         """
-        return super(ConfigResource, self).get(identifier)
+        return super().get(identifier)
 
-    def create_object(self, obj):
+    def _create_object(self, spec, parent, share_with, metakeys):
         try:
             return Config.get_or_create(
-                obj.data["cfg"],
-                obj.data["family"],
-                obj.data.get("config_type", "static")
+                spec.data["cfg"],
+                spec.data["family"],
```

Comment:
lack of typings makes this look like it could cause `KeyError`, while docs don't say this could `raise` anything

Reply:
Well, we don't use pydantic so we must assume that it's deserialized object following the ConfigCreateSpecSchema